sc/src/main/javascript/bundles/feedback-form/html/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
```

Comment:
Seems like you're duplicating this from the component repository itself. I don't think it's necessary, given it's such a simple component. You can remove the `html` folder, along with the webpack and babel settings. You can take inspiration from the `autocomplete` bundle, as it's equally simple. Same comments apply for gxa.